src/main/java/codechicken/multipart/api/part/ModelRenderPart.java

```diff
@@ -3,20 +3,20 @@
 import codechicken.multipart.api.part.render.PartBakedModelRenderer;
 import net.minecraft.client.renderer.RenderType;
 import net.minecraft.world.level.block.state.BlockState;
-import net.minecraftforge.client.model.data.EmptyModelData;
-import net.minecraftforge.client.model.data.IModelData;
+import net.minecraftforge.client.model.data.ModelData;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * Companion to {@link PartBakedModelRenderer}
  */
 public interface ModelRenderPart extends MultiPart {
 
-    boolean canRenderInLayer(@Nullable RenderType layer);
+    @Deprecated(forRemoval = true, since = "1.19.2") // Handled by BakedModel directly now
+    default boolean canRenderInLayer(@Nullable RenderType layer) { return false; }
```

Comment:
Breaking changes window. This can just be completely nuked instead of deprecated.